### PR TITLE
[quant][refactor tests] Clean up test_quantization.py

### PR DESCRIPTION
**Pull request by supriyar**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #59089 [quant][refactor tests] Move qtensor serialization tests from test_deprecated_jit
* **#59088 [quant][refactor tests] Clean up test_quantization.py**

Summary:
Clean up comments and organize the tests better

Test Plan:
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D28750064](https://our.internmc.facebook.com/intern/diff/D28750064)

### CODE REVIEW

**Comment by supriyar** [comment]
@supriyar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28750064).

File: test/test_quantization.py [hunk 1]
```diff
@@ -2,10 +2,14 @@
...
+# - quantized kernels
+# - quantized functional operators
+# - quantized workflow modules
+# - quantized workflow operators
+# - quantized tensor
+
+# 1. Quantized Kernels
 from quantization.core.test_quantized_op import TestQuantizedOps  # noqa: F401
 from quantization.core.test_quantized_op import TestQNNPackOps  # noqa: F401
 from quantization.core.test_quantized_op import TestQuantizedLinear  # noqa: F401
```
Comment (jerryzh168) [inline test/test_quantization.py:15]: I feel ideally we should merge these into TestQuantizedOps actually.

**Review by jerryzh168 — APPROVED** [review]
Looks good, had one comment that can be addressed in a future PR.

File: test/test_quantization.py (hunk 1 above)
Comment (supriyar) [reply test/test_quantization.py:15]: Yeah I think we can have a unified class for all ops. I'll add a todo

**Comment by supriyar** [comment]
@supriyar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28750064).

**Pull request closed without merging**